hw/ip/otbn/README.md

```diff
@@ -13,11 +13,37 @@ through the [GitHub issue tracker](https://github.com/lowRISC/opentitan/issues).
 
 ## Develop OTBN
 
-### Build the RTL implementation
+### Run the standalone simulation
 
-To build the RTL implementation of OTBN in a simulation, run `fusesoc` without
-passing the `OTBN_MODEL` flag. For example, the Verilator simulation can be
-compiled as follows.
+*Note that OTBN is still in the early development stages so this simulation will
+do very little currently*
+
+A standalone environment to run OTBN alone in verilator is included. It is built
+with `fusesoc`:
+
+```sh
+fusesoc --cores-root=. run --target=sim --setup --build lowrisc:ip:otbn_top_sim
+```
+
+It includes functionality to set the initial DMem and IMem contents with a .vmem
+or .elf file. The start address is hard coded to 0. Modify the `ImemStartAddr`
```

Comment:
Probably drop the reference to ELF files? A `.elf` suffix is non-standard, and we don't support a single ELF "out of the box" anyway at the moment. And the example below uses `.vmem` files anyway.